test/cmdlineTests/optimizer_array_sload/output

```diff
@@ -0,0 +1,83 @@
+Optimized IR:
```

Comment:
[Diff](https://gist.github.com/hrkrshnn/5d054ed71e068fbd02e9ead9599a6af9) to illustrate the change.